File: backend/templates/generator.py
import random

def generate_test():
    # Example generator in Python - modify as needed
    n = random.randint(1, 100)
    m = random.randint(1, 100)
    
    print(n, m)
    
    numbers = [random.randint(1, 1000) for _ in range(n)]
    print(' '.join(map(str, numbers)))

if __name__ == "__main__":
    generate_test()

File: backend/templates/generator.cpp
#include "testlib.h"
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main(int argc, char* argv[]) {
    registerGen(argc, argv, 1);
    
    // Example generator - modify as needed
    int n = rnd.next(1, 100);  // Random number from 1 to 100
    int m = rnd.next(1, 100);  // Random number from 1 to 100
    
    cout << n << " " << m << endl;
    
    for (int i = 0; i < n; i++) {
        cout << rnd.next(1, 1000);  // Random number from 1 to 1000
        if (i < n - 1) cout << " ";
    }
    cout << endl;
    
    return 0;
}
